osu.Game/Online/Leaderboards/LeaderBoardScoreV2.cs

```diff
@@ -0,0 +1,550 @@
+                : base(content, timeBeforeLoad)
+            {
+                CornerRadius = corner_radius;
+                Masking = true;
+            }
+        }
+
+        private partial class ColouredModSwitchTiny : ModSwitchTiny, IHasTooltip
+        {
+            private readonly IMod mod;
+
+            [Resolved]
+            private OsuColour colours { get; set; } = null!;
+
+            public ColouredModSwitchTiny(IMod mod)
+                : base(mod)
+            {
+                this.mod = mod;
+                Masking = true;
+                EdgeEffect = new EdgeEffectParameters
+                {
+                    Roundness = 15,
+                    Type = EdgeEffectType.Shadow,
+                    Colour = Colour4.Black.Opacity(0.15f),
+                    Radius = 3,
+                    Offset = new Vector2(-2, 0)
+                };
+            }
+
+            protected override void UpdateState()
+            {
+                AcronymText.Colour = Colour4.FromHex("#555555");
+                Background.Colour = colours.Yellow;
+            }
+
+            public virtual LocalisableString TooltipText => (mod as Mod)?.IconTooltip ?? mod.Name;
+        }
+
+        #endregion
+
+        public MenuItem[] ContextMenuItems
+        {
+            get
+            {
+                List<MenuItem> items = new List<MenuItem>();
+
+                if (score.Mods.Length > 0 && modsContainer.Any(s => s.IsHovered) && songSelect != null)
+                    items.Add(new OsuMenuItem("Use these mods", MenuItemType.Highlighted, () => songSelect.Mods.Value = score.Mods));
+
+                if (score.Files.Count <= 0) return items.ToArray();
+
+                items.Add(new OsuMenuItem("Export", MenuItemType.Standard, () => new LegacyScoreExporter(storage).Export(score)));
+                items.Add(new OsuMenuItem(CommonStrings.ButtonsDelete, MenuItemType.Destructive, () => dialogOverlay?.Push(new LocalScoreDeleteDialog(score))));
+
+                return items.ToArray();
+            }
+        }
+
+        private LocalisableString getResults(ScoreInfo score)
+        {
+            string resultString = score.GetStatisticsForDisplay().Where(s => s.Result.IsBasic()).Aggregate(string.Empty, (current, result) =>
+                current.Insert(current.Length, $"{result.Count}/"));
+
+            return resultString.Remove(resultString.Length - 1);
+        }
+
+        private LocalisableString getResultNames(ScoreInfo score)
+        {
+            string resultName = string.Empty;
+
+            foreach (var hitResult in score.GetStatisticsForDisplay().Where(s => s.Result.IsBasic()))
+            {
+                switch (hitResult.Result)
+                {
+                    case HitResult.Perfect:
+                        appendToString("320/");
+                        break;
+
+                    case HitResult.Great:
+                        appendToString("300/");
```

Comment:
Also ignore the design here, judgements in lazer are not numbers.

Reply:
alr